fenix/app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -1762,4 +1762,9 @@ class Settings(private val appContext: Context) : PreferencesHolder {
         key = appContext.getPreferenceKey(R.string.pref_key_growth_early_search),
         default = false,
     )
+
+    /**
+     * Indicates if the new Search settings UI is enabled.
+     */
+    var enableNewSearchSettings: Boolean = showUnifiedSearchFeature && FeatureFlags.newSearchSettings
```

Comment:
I have the same nit about the `NewSearchSettings` naming here. Would appreciate it if we could find something slightly more descriptive

Reply:
Changed it to `enableNewSearchSettingsUI`, to say roughly the same as the comment. I can't come up with a better naming, but open for suggestions :)